tools/pythonpkg/tests/fast/arrow/test_arrow_fetch_recordbatch.py

```diff
@@ -106,3 +106,15 @@ def test_record_coverage(self, duckdb_cursor):
         chunk = record_batch_reader.read_all()
         assert(len(chunk) == 2048)
 
+    def test_record_batch_query_error(self, duckdb_cursor):
+        if not can_run:
+            return
+        duckdb_cursor = duckdb.connect()
+        duckdb_cursor.execute("CREATE table t as select 'foo' as a;")
+        query = duckdb_cursor.execute("SELECT cast(a as double) FROM t")
+        record_batch_reader = query.fetch_record_batch(1024)
+        try:
+            chunk = record_batch_reader.read_next_batch()
+            assert(False)
+        except Exception as e:
+            assert('Conversion Error' in str(e))
```

Comment:
```suggestion
        with pytest.raises(RuntimeError, match='Conversion Error'):
            record_batch_reader.read_next_batch()
```

Reply:
Not sure where it arises but in this case the exception is of type `OSError`. Test should be fixed now.